package test.action; 

import org.junit.Test; 
import org.junit.Before; 
import org.junit.After; 

/** 
* Register Tester. 
* 
* @author <Authors name> 
* @since <pre>6 30, 2019</pre>
* @version 1.0 
*/ 
public class RegisterTest { 

@Before
public void before() throws Exception { 
} 

@After
public void after() throws Exception { 
} 

/** 
* 
* Method: getTelephone() 
* 
*/ 
@Test
public void testGetTelephone() throws Exception { 
//TODO: Test goes here... 
} 

/** 
* 
* Method: setTelephone(String telephone) 
* 
*/ 
@Test
public void testSetTelephone() throws Exception { 
//TODO: Test goes here... 
} 

/** 
* 
* Method: getPassword1() 
* 
*/ 
@Test
public void testGetPassword1() throws Exception { 
//TODO: Test goes here... 
} 

/** 
* 
* Method: setPassword1(String password1) 
* 
*/ 
@Test
public void testSetPassword1() throws Exception { 
//TODO: Test goes here... 
} 

/** 
* 
* Method: getPassword2() 
* 
*/ 
@Test
public void testGetPassword2() throws Exception { 
//TODO: Test goes here... 
} 

/** 
* 
* Method: setPassword2(String password2) 
* 
*/ 
@Test
public void testSetPassword2() throws Exception { 
//TODO: Test goes here... 
} 

/** 
* 
* Method: getUserService() 
* 
*/ 
@Test
public void testGetUserService() throws Exception { 
//TODO: Test goes here... 
} 

/** 
* 
* Method: setUserService(UserService userService) 
* 
*/ 
@Test
public void testSetUserService() throws Exception { 
//TODO: Test goes here... 
} 

/** 
* 
* Method: getUser() 
* 
*/ 
@Test
public void testGetUser() throws Exception { 
//TODO: Test goes here... 
} 

/** 
* 
* Method: setUser(User user) 
* 
*/ 
@Test
public void testSetUser() throws Exception { 
//TODO: Test goes here... 
} 

/** 
* 
* Method: execute() 
* 
*/ 
@Test
public void testExecute() throws Exception { 
//TODO: Test goes here... 
} 


} 
